src/containers/reports/export/ExportBulkReducer.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import {
+  List,
+  Map,
+  fromJS,
+} from 'immutable';
+import { RequestStates } from 'redux-reqseq';
+
+import { RESET_EXPORT_CRISIS_XML, exportCrisisXMLByDateRange } from './ExportActions';
+
+const INITIAL_STATE :Map = fromJS({
+  errors: List(),
+  fetchState: RequestStates.STANDBY,
```

Comment:
not necessary but you could rename `fetchState` to the action name itself, so that you can easily use `useRequestState` from `lattice-utils` to read that value in your component:

```
[EXPORT_CRISIS_XML_BY_DATE_RANGE]: { REQUEST_STATE: RequestStates.STANDBY }